### ISSUE TITLE
[Feature Request] Allow to create inventory source from a manual project

### ISSUE THREAD

**Issue by jean-christophe-manciot** [ansible/awx#1823]
##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME
 - web

##### SUMMARY
As of 1.0.6, it is not possible to import a local hosts file and use it as an inventory.

##### ENVIRONMENT
* AWX version: tag 1.0.6
* AWX install method: docker/docker-compose on linux
* Ansible version: v2.5.2 (from PyPi)
* Operating System: Ubuntu 17.10
* docker-ce ```18.04.0~ce~3-0~ubuntu``` (from official docker PPA)
* docker==3.3.0
* docker-compose==1.21.1
* requests==2.18.1

##### STEPS TO REPRODUCE

- Add a project folder below the projects base folder containing a hosts file such as:
```
[all:vars]
nms_mgt_ip_address=172.21.0.1
remote_domain_name=actionmystique.net

[firewalls]
ASAv1 ansible_host=172.21.154.231

[spines]
NX_OSv_9k_Spine_31 ansible_host=172.21.114.153 
NX_OSv_9k_Spine_32 ansible_host=172.21.114.153

[mgt]
IOSv_L2_10 ansible_host=172.21.100.30

[nx_osv_9k:children]
spines

[asav:children]
firewalls

[cloud:children]
firewalls
spines
mgt
```
- Add this manual project in the GUI with Projects/+ADD.
- Add inventory in the GUI with Inventories/+ADD
- Add source --> Sourced from a project
- Choose an inventory file --> /(project root)

##### EXPECTED RESULTS
It should be possible to import a local hosts file and use it as an inventory.

##### ACTUAL RESULTS
![error loading hosts file from manual project](https://user-images.githubusercontent.com/13176858/39424242-f036cfe4-4c75-11e8-8dd5-24f7ab0b8220.png)

**Comment by AlanCoding** [comment]
I don't think this would be a major problem to allow, with simple qualifiers:

 - we will not try to detect changes in the source tree on the file system
 - user will be responsible for defining the update behavior on their own (like Update on Launch)

**Comment by PsychoSid** [comment]
Did this not used to be allowed ? Sure I used this prior to v1.0.5.

**Comment by AlanCoding** [comment]
I don't think so, but I don't think it would have worked. There is more than 1 place where we need to add some special `if` clauses, because ordinarily we need a project sync before running the inventory update, and in this case we specifically need to not do that.

**Comment by jacobwoffenden** [comment]
Is there any update on this request? I've just been caught out by it.

**Comment by Yinaij** [comment]
Also been caught by this one. Got a case where I have AWX running in a closed system. Can't have any outbound connections to fetch the hosts from. Thought it would be possible to keep both project and inventory locally, but it seems I was wrong.

Would appreciate an update on this issue.
